plugins/heroku/src/tasks/production.ts

```diff
@@ -20,6 +21,25 @@ export default class HerokuProduction extends Task<typeof HerokuSchema> {
       await setSlug(slugId)
 
       console.log(`staging has been successfully promoted to production`)
+
+      if (this.options.scale) {
+        const { scale } = this.options
+        console.log(`scaling to ${scale} processes...`)
+
+        console.log('retrieving production app ID...')
+        const state = readState('production')
+        if (!state) {
+          throw new ToolKitError('could not find production state information')
+        }
+        const { appIds } = state
+
+        if (appIds.length > 1) {
+          console.warn('tool kit can currently only scale your first app')
```

Comment:
Question: why are we only scaling the first app? A lot of apps will be in both eu and us.?